include/itkLogNormalDistributionImageSource.hxx

```diff
@@ -87,7 +69,7 @@ LogNormalDistributionImageSource< TImage >
     {
     while( !it.IsAtEndOfLine() )
       {
-      it.Set( logNormalDistribution( generator ) );
+      it.Set( normalGenerator->GetVariate() );
```

Comment:
Nice modification :+1: 

Since we now are generating a normal distribution instead of a log normal distribution, let's change the class name from `LogNormalDistributionImageSource` to `NormalDistributionImageSource`.